cluster/charts/rook-ceph/values.yaml

```diff
@@ -133,6 +134,7 @@ hostpathRequiresPrivileged: false
 
 # Disable automatic orchestration when new devices are discovered.
 disableDeviceHotplug: false
+discoverDaemonUdev: 
```

Comment:
Needs doc comment.

Reply:
Okay

Reply:
An example should also be provided as well so users know how to use the option.